source/fundamentals/indexes.txt

```diff
@@ -128,23 +128,22 @@ To learn more, see  :manual:`Compound Indexes </core/index-compound>`.
 Multikey Indexes (Indexes on Array Fields)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**Multikey indexes** are indexes that improve performance for queries that
-specify ascending or descending indexes on fields that contain an array value.
-You can define a multikey index using the same syntax as a single field or
-compound index.
+**Multikey indexes** are indexes that improve performance for queries on
+fields that contain array values. You can define a multikey index using
+the same syntax as when you create a single field or compound index.
 
-The following example use the ``createIndex()`` method to create an ascending
-index on the ``cast`` field (array of names) in the ``movies`` collection in
-the ``sample_mflix`` database.
+The following example uses the ``createIndex()`` method to create an ascending
+index on the ``cast`` field, which contains an array of names, in the
+``movies`` collection of the ``sample_mflix`` database.
```

Comment:
Suggestion:

From reading the current content, I think introducing the method could be separated out into its own sentence rather than combining it with a description of the example. Seems like this uses the same pattern as the Single Field Index section, but maybe that should be separated as well.

Personal preference, but I try to avoid separating out non-restrictive phrases with commas or parentheses in docs because it slows down my own reading speed. I wasn't able to think of a good way to phrase this, so I left it to the reader to infer the cast field is an array of values in the following suggested edit:

"You can create a multikey index on a field that contains an array of values by calling the createIndex() method. The following example shows how you can create an ascending index on the cast field in the movies collection of the sample_mflix database".